modules/andes-core/client/src/main/java/org/wso2/andes/XaJmsSession.java

```diff
@@ -277,6 +278,62 @@ public void unsubscribe(String s) throws JMSException {
         session.unsubscribe(s);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public MessageConsumer createSharedConsumer(Topic topic, String sharedSubscriptionName)
```

Comment:
Create jira for un implimented methods ?

Reply:
Created https://github.com/wso2/product-ei/issues/1168.